### PR TITLE
Use cascoda config by default

### PR DESCRIPTION
**Pull request by nikhil-zinjurde-imgtec**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,9 +92,10 @@ We are maintaining different pre-defined configurations in "config" files.
     creator-platform-all.config - Basic Creator Platform profile enabling all the userspace and kernel space packages suitable for CC2520 based platforms.
     creator-platform-all-cascoda.config - Basic Creator Platform profile enabling all the userspace and kernel space packages suitable for Cascoda CA8210 based platforms.
 
-## To build for cascoda CA8210 platform:
+## To build for Cascoda CA8210 platform:
+
+_Note :- By default, CreatorKit project1 for Cascoda CA8210 platform is built if no config is specific using P= option._
```
Comment (abhijit-mahajani-imgtec) [inline README.md]: may be good to rephrase to:

By default, CreatorKit project1 is built for Cascoda CA8210 platform, if no config has been specified using P= option

**Comment by abhijit-mahajani-imgtec** [comment]
Approved

**Comment by abhijit-mahajani-imgtec** [comment]
Everything getting built fine but getting warning:

find /home/abhijit/work/github-creator-kit-dev/build/../dist/openwrt/bin/pistachio/ -maxdepth 1 -type f -exec cp {} /home/abhijit/work/github-creator-kit-dev/build/output/openwrt/ \;
echo "OpenWrt Done!"
OpenWrt Done!
/bin/sh: line 0: [: -eq: unary operator expected
/bin/sh: line 0: [: -eq: unary operator expected
mkdir -p /home/abhijit/work/github-creator-kit-dev/build/output/contiki
/bin/sh: line 0: [: -eq: unary operator expected
/bin/sh: line 0: [: -eq: unary operator expected
echo "Contiki Done!"
Contiki Done!
echo "All Done!"
All Done!

**Comment by nikhil-zinjurde-imgtec** [comment]
That's because ideally it expects the P option to be used

**Pull request merged**